spec/controllers/api/v1/authentication_controller_spec.rb

```diff
@@ -8,6 +8,7 @@
   let(:pin)    { user.most_recent_token[:otp_code] }
   let(:mobile) { generate(:mobile) }
   let(:mobile1) { generate(:mobile) }
+  let(:district_id) { create(:district).id.to_s }
```

Comment:
do we need the `to_s` bit?
```suggestion
  let(:district_id) { create(:district).id }
```

Reply:
it was just for comparison to the json output, which comes as a string